panda/src/configfiles/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Set some variables that are used in panda.prc.in.
+set(AUX_DISPLAYS)
+
+if(HAVE_GL)
+  set(AUX_DISPLAYS "${AUX_DISPLAYS}\naux-display pandagl")
+endif()
+
+if(HAVE_DX9)
+  set(AUX_DISPLAYS "${AUX_DISPLAYS}\naux-display pandadx9")
+endif()
+
+if(HAVE_TINYDISPLAY)
+  set(AUX_DISPLAYS "${AUX_DISPLAYS}\naux-display p3tinydisplay")
+endif()
+
+if(HAVE_OPENAL)
+  set(AUDIO_LIBRARY_NAME p3openal_audio)
+elseif(HAVE_FMODEX)
+  set(AUDIO_LIBRARY_NAME p3fmod_audio)
+else()
+  set(AUDIO_LIBRARY_NAME none)
+endif()
+
+configure_file(panda.prc.in "${CMAKE_BINARY_DIR}/etc/20_panda.prc")
+if(IS_MULTICONFIG)
+  file(GENERATE OUTPUT "${PROJECT_BINARY_DIR}/$<CONFIG>/etc/20_panda.prc"
+                INPUT "${PROJECT_BINARY_DIR}/etc/20_panda.prc")
+endif()
+
+if(WIN32)
+  set(_confdir "etc")
+
+else()
+  # On Unices, ask the GNUInstallDirs module where /etc is
+  set(_confdir "${CMAKE_INSTALL_FULL_SYSCONFDIR}")
+
+endif()
+
+install(FILES "${CMAKE_BINARY_DIR}/etc/20_panda.prc" COMPONENT Core DESTINATION ${_confdir})
```

Comment:
I am fine with changing the prc files to this scheme, however, if we make this step I think we should simultaneously also install the etc files to `/etc/panda3d` instead of `/etc` (and start respecting the XDG specification).

We can probably also better split them up, eg. the egg specific stuff should probably be in its own file.

Reply:
I like all points here. Installing to `/etc/panda3d` sounds great - would the search path for that be `<auto>/etc/panda3d`?

What do we use as the directory on Windows? What path do they reside under when running Panda from the build directory (i.e. not installed)?

Reply:
I think for system-wide installations, we search `$XDG_CONFIG_HOME/panda3d` and `$XDG_CONFIG_DIRS/panda3d` without going up the filesystem specifically.  Although, looking at the [spec](https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html), the latter appears to be `/etc/xdg` by default, which I'm not 100% sure people are familiar with.

On Windows and non-systemwide installations we can use the current behaviour of having it under an `etc` directory in the build directory (or under `panda3d/etc` for a pip installation).